content/06.know-your-problem.md

```diff
@@ -28,7 +28,9 @@ If one assumes that all the samples are independent but that is in fact not the
 
 In general, deep learning has an increased tendency for overfitting, compared to classical methods, due to the large number of parameters being estimated, making issues of adequate sample size even more important (see [Tip 7](#overfitting)).
 For a large dataset, overfitting may not be a concern, but the modeling power of deep learning may lead to more spurious correlations and thus incorrect interpretation of results (see [Tip 9](#interpretation)).
-Finally, it is important to note that with the exception of very specific cases of unsupervised data analysis, it is generally the case that a molecular or imaging dataset does not have much value without appropriate clinical or demographic data; this must always be balanced with the need to protect patient privacy (see [Tip 10](#privacy)).
+It is important to note that often molecular or imaging datasets require appropriate clinical or demographic data to support robust analyses; this must always be balanced with the need to protect patient privacy (see [Tip 10](#privacy)).
 Looking at these annotations can also clarify the study design (for example, by seeing if all the individuals are adolescents or women) or at least help the analyst employing deep learning to know what questions to ask.
+One tool in the practicioners toolkit for dealing with datasets that are not thoroughly understood is data simulation.

```

Comment:
I think this should be "practitioner's" not "practitioners." Sounds great otherwise.

Reply:
Thanks @rasbt! Fixed it.